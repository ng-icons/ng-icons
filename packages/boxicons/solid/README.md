# @ng-icons/boxicons/solid

Secondary entry point of `@ng-icons/boxicons`. It can be used by importing from `@ng-icons/boxicons/solid`.
