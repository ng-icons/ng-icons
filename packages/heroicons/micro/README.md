# @ng-icons/heroicons/micro

Secondary entry point of `@ng-icons/heroicons`. It can be used by importing from `@ng-icons/heroicons/micro`.
