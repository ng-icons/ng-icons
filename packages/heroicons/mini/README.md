# @ng-icons/heroicons/mini

Secondary entry point of `@ng-icons/heroicons`. It can be used by importing from `@ng-icons/heroicons/mini`.
