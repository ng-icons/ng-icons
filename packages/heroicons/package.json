{
  "name": "@ng-icons/heroicons",
  "version": "0.0.1",
  "license": "MIT",
  "repository": {
    "url": "https://github.com/ng-icons/ng-icons"
  },
  "homepage": "https://ng-icons.github.io/ng-icons/",
  "peerDependencies": {
    "@angular/common": ">=14.0.0",
    "@angular/core": ">=14.0.0"
  },
  "dependencies": {
    "tslib": "^2.2.0"
  }
}
