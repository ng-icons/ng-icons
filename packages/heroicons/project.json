{
  "name": "heroicons",
  "$schema": "../../node_modules/nx/schemas/project-schema.json",
  "projectType": "library",
  "sourceRoot": "packages/heroicons/src",
  "prefix": "ng-icons",
  "targets": {
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/packages/heroicons"],
      "options": {
        "jestConfig": "packages/heroicons/jest.config.ts",
        "passWithNoTests": true
      }
    },
    "build": {
      "executor": "@nx/angular:package",
      "outputs": ["{workspaceRoot}/dist/packages/heroicons"],
      "options": {
        "project": "packages/heroicons/ng-package.json",
        "updateBuildableProjectDepsInPackageJson": true
      },
      "configurations": {
        "production": {
          "tsConfig": "packages/heroicons/tsconfig.lib.prod.json"
        },
        "development": {
          "tsConfig": "packages/heroicons/tsconfig.lib.json"
        }
      },
      "defaultConfiguration": "production"
    },
    "lint": {
      "executor": "@nx/linter:eslint",
      "options": {
        "lintFilePatterns": [
          "packages/heroicons/src/**/*.ts",
          "packages/heroicons/src/**/*.html",
          "packages/heroicons/outline/**/*.ts",
          "packages/heroicons/outline/**/*.html",
          "packages/heroicons/solid/**/*.ts",
          "packages/heroicons/solid/**/*.html",
          "packages/heroicons/mini/**/*.ts",
          "packages/heroicons/mini/**/*.html"
        ]
      },
      "outputs": ["{options.outputFile}"]
    },
    "deploy": {
      "executor": "ngx-deploy-npm:deploy",
      "options": {
        "access": "public"
      }
    }
  },
  "tags": []
}
