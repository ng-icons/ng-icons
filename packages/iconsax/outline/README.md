# @ng-icons/iconsax/outline

Secondary entry point of `@ng-icons/iconsax`. It can be used by importing from `@ng-icons/iconsax/outline`.
