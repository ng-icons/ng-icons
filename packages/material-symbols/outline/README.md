# @ng-icons/material-symbols/outline

Secondary entry point of `@ng-icons/material-symbols`. It can be used by importing from `@ng-icons/material-symbols/outline`.
