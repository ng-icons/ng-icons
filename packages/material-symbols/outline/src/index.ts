export const matOutlinedOutline = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 -960 960 960" fill="currentColor"><path d="M796-121 533-384q-30 26-69.96 40.5Q423.08-329 378-329q-108.16 0-183.08-75Q120-479 120-585t75-181q75-75 181.5-75t181 75Q632-691 632-584.85 632-542 618-502q-14 40-42 75l264 262-44 44ZM377-389q81.25 0 138.13-57.5Q572-504 572-585t-56.87-138.5Q458.25-781 377-781q-82.08 0-139.54 57.5Q180-666 180-585t57.46 138.5Q294.92-389 377-389ZM275-556v-60h201v60H275Z"></path></svg>`;
