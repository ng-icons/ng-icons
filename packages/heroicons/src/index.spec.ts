import * as microIcons from '@ng-icons/heroicons/micro';
import * as miniIcons from '@ng-icons/heroicons/mini';
import * as outlineIcons from '@ng-icons/heroicons/outline';
import * as solidIcons from '@ng-icons/heroicons/solid';

describe('Heroicons Icons', () => {
  it('should ensure the export names have not changed unexpectedly', () => {
    expect(Object.keys(outlineIcons)).toMatchSnapshot();
    expect(Object.keys(solidIcons)).toMatchSnapshot();
    expect(Object.keys(microIcons)).toMatchSnapshot();
    expect(Object.keys(miniIcons)).toMatchSnapshot();
  });
});
