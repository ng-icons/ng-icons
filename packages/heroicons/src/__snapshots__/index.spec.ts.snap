// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heroicons Icons should ensure the export names have not changed unexpectedly 1`] = `
Array [
  "heroAcademicCap",
  "heroAdjustmentsHorizontal",
  "heroAdjustmentsVertical",
  "heroArchiveBoxArrowDown",
  "heroArchiveBoxXMark",
  "heroArchiveBox",
  "heroArrowDownCircle",
  "heroArrowDownLeft",
  "heroArrowDownOnSquareStack",
  "heroArrowDownOnSquare",
  "heroArrowDownRight",
  "heroArrowDownTray",
  "heroArrowDown",
  "heroArrowLeftCircle",
  "heroArrowLeftEndOnRectangle",
  "heroArrowLeftOnRectangle",
  "heroArrowLeftStartOnRectangle",
  "heroArrowLeft",
  "heroArrowLongDown",
  "heroArrowLongLeft",
  "heroArrowLongRight",
  "heroArrowLongUp",
  "heroArrowPathRoundedSquare",
  "heroArrowPath",
  "heroArrowRightCircle",
  "heroArrowRightEndOnRectangle",
  "heroArrowRightOnRectangle",
  "heroArrowRightStartOnRectangle",
  "heroArrowRight",
  "heroArrowSmallDown",
  "heroArrowSmallLeft",
  "heroArrowSmallRight",
  "heroArrowSmallUp",
  "heroArrowTopRightOnSquare",
  "heroArrowTrendingDown",
  "heroArrowTrendingUp",
  "heroArrowUpCircle",
  "heroArrowUpLeft",
  "heroArrowUpOnSquareStack",
  "heroArrowUpOnSquare",
  "heroArrowUpRight",
  "heroArrowUpTray",
  "heroArrowUp",
  "heroArrowUturnDown",
  "heroArrowUturnLeft",
  "heroArrowUturnRight",
  "heroArrowUturnUp",
  "heroArrowsPointingIn",
  "heroArrowsPointingOut",
  "heroArrowsRightLeft",
  "heroArrowsUpDown",
  "heroAtSymbol",
  "heroBackspace",
  "heroBackward",
  "heroBanknotes",
  "heroBars2",
  "heroBars3BottomLeft",
  "heroBars3BottomRight",
  "heroBars3CenterLeft",
  "heroBars3",
  "heroBars4",
  "heroBarsArrowDown",
  "heroBarsArrowUp",
  "heroBattery0",
  "heroBattery100",
  "heroBattery50",
  "heroBeaker",
  "heroBellAlert",
  "heroBellSlash",
  "heroBellSnooze",
  "heroBell",
  "heroBoltSlash",
  "heroBolt",
  "heroBookOpen",
  "heroBookmarkSlash",
  "heroBookmarkSquare",
  "heroBookmark",
  "heroBriefcase",
  "heroBugAnt",
  "heroBuildingLibrary",
  "heroBuildingOffice2",
  "heroBuildingOffice",
  "heroBuildingStorefront",
  "heroCake",
  "heroCalculator",
  "heroCalendarDays",
  "heroCalendar",
  "heroCamera",
  "heroChartBarSquare",
  "heroChartBar",
  "heroChartPie",
  "heroChatBubbleBottomCenterText",
  "heroChatBubbleBottomCenter",
  "heroChatBubbleLeftEllipsis",
  "heroChatBubbleLeftRight",
  "heroChatBubbleLeft",
  "heroChatBubbleOvalLeftEllipsis",
  "heroChatBubbleOvalLeft",
  "heroCheckBadge",
  "heroCheckCircle",
  "heroCheck",
  "heroChevronDoubleDown",
  "heroChevronDoubleLeft",
  "heroChevronDoubleRight",
  "heroChevronDoubleUp",
  "heroChevronDown",
  "heroChevronLeft",
  "heroChevronRight",
  "heroChevronUpDown",
  "heroChevronUp",
  "heroCircleStack",
  "heroClipboardDocumentCheck",
  "heroClipboardDocumentList",
  "heroClipboardDocument",
  "heroClipboard",
  "heroClock",
  "heroCloudArrowDown",
  "heroCloudArrowUp",
  "heroCloud",
  "heroCodeBracketSquare",
  "heroCodeBracket",
  "heroCog6Tooth",
  "heroCog8Tooth",
  "heroCog",
  "heroCommandLine",
  "heroComputerDesktop",
  "heroCpuChip",
  "heroCreditCard",
  "heroCubeTransparent",
  "heroCube",
  "heroCurrencyBangladeshi",
  "heroCurrencyDollar",
  "heroCurrencyEuro",
  "heroCurrencyPound",
  "heroCurrencyRupee",
  "heroCurrencyYen",
  "heroCursorArrowRays",
  "heroCursorArrowRipple",
  "heroDevicePhoneMobile",
  "heroDeviceTablet",
  "heroDocumentArrowDown",
  "heroDocumentArrowUp",
  "heroDocumentChartBar",
  "heroDocumentCheck",
  "heroDocumentDuplicate",
  "heroDocumentMagnifyingGlass",
  "heroDocumentMinus",
  "heroDocumentPlus",
  "heroDocumentText",
  "heroDocument",
  "heroEllipsisHorizontalCircle",
  "heroEllipsisHorizontal",
  "heroEllipsisVertical",
  "heroEnvelopeOpen",
  "heroEnvelope",
  "heroExclamationCircle",
  "heroExclamationTriangle",
  "heroEyeDropper",
  "heroEyeSlash",
  "heroEye",
  "heroFaceFrown",
  "heroFaceSmile",
  "heroFilm",
  "heroFingerPrint",
  "heroFire",
  "heroFlag",
  "heroFolderArrowDown",
  "heroFolderMinus",
  "heroFolderOpen",
  "heroFolderPlus",
  "heroFolder",
  "heroForward",
  "heroFunnel",
  "heroGif",
  "heroGiftTop",
  "heroGift",
  "heroGlobeAlt",
  "heroGlobeAmericas",
  "heroGlobeAsiaAustralia",
  "heroGlobeEuropeAfrica",
  "heroHandRaised",
  "heroHandThumbDown",
  "heroHandThumbUp",
  "heroHashtag",
  "heroHeart",
  "heroHomeModern",
  "heroHome",
  "heroIdentification",
  "heroInboxArrowDown",
  "heroInboxStack",
  "heroInbox",
  "heroInformationCircle",
  "heroKey",
  "heroLanguage",
  "heroLifebuoy",
  "heroLightBulb",
  "heroLink",
  "heroListBullet",
  "heroLockClosed",
  "heroLockOpen",
  "heroMagnifyingGlassCircle",
  "heroMagnifyingGlassMinus",
  "heroMagnifyingGlassPlus",
  "heroMagnifyingGlass",
  "heroMapPin",
  "heroMap",
  "heroMegaphone",
  "heroMicrophone",
  "heroMinusCircle",
  "heroMinusSmall",
  "heroMinus",
  "heroMoon",
  "heroMusicalNote",
  "heroNewspaper",
  "heroNoSymbol",
  "heroPaintBrush",
  "heroPaperAirplane",
  "heroPaperClip",
  "heroPauseCircle",
  "heroPause",
  "heroPencilSquare",
  "heroPencil",
  "heroPhoneArrowDownLeft",
  "heroPhoneArrowUpRight",
  "heroPhoneXMark",
  "heroPhone",
  "heroPhoto",
  "heroPlayCircle",
  "heroPlayPause",
  "heroPlay",
  "heroPlusCircle",
  "heroPlusSmall",
  "heroPlus",
  "heroPower",
  "heroPresentationChartBar",
  "heroPresentationChartLine",
  "heroPrinter",
  "heroPuzzlePiece",
  "heroQrCode",
  "heroQuestionMarkCircle",
  "heroQueueList",
  "heroRadio",
  "heroReceiptPercent",
  "heroReceiptRefund",
  "heroRectangleGroup",
  "heroRectangleStack",
  "heroRocketLaunch",
  "heroRss",
  "heroScale",
  "heroScissors",
  "heroServerStack",
  "heroServer",
  "heroShare",
  "heroShieldCheck",
  "heroShieldExclamation",
  "heroShoppingBag",
  "heroShoppingCart",
  "heroSignalSlash",
  "heroSignal",
  "heroSparkles",
  "heroSpeakerWave",
  "heroSpeakerXMark",
  "heroSquare2Stack",
  "heroSquare3Stack3d",
  "heroSquares2x2",
  "heroSquaresPlus",
  "heroStar",
  "heroStopCircle",
  "heroStop",
  "heroSun",
  "heroSwatch",
  "heroTableCells",
  "heroTag",
  "heroTicket",
  "heroTrash",
  "heroTrophy",
  "heroTruck",
  "heroTv",
  "heroUserCircle",
  "heroUserGroup",
  "heroUserMinus",
  "heroUserPlus",
  "heroUser",
  "heroUsers",
  "heroVariable",
  "heroVideoCameraSlash",
  "heroVideoCamera",
  "heroViewColumns",
  "heroViewfinderCircle",
  "heroWallet",
  "heroWifi",
  "heroWindow",
  "heroWrenchScrewdriver",
  "heroWrench",
  "heroXCircle",
  "heroXMark",
]
`;

exports[`Heroicons Icons should ensure the export names have not changed unexpectedly 2`] = `
Array [
  "heroAcademicCapSolid",
  "heroAdjustmentsHorizontalSolid",
  "heroAdjustmentsVerticalSolid",
  "heroArchiveBoxArrowDownSolid",
  "heroArchiveBoxXMarkSolid",
  "heroArchiveBoxSolid",
  "heroArrowDownCircleSolid",
  "heroArrowDownLeftSolid",
  "heroArrowDownOnSquareStackSolid",
  "heroArrowDownOnSquareSolid",
  "heroArrowDownRightSolid",
  "heroArrowDownTraySolid",
  "heroArrowDownSolid",
  "heroArrowLeftCircleSolid",
  "heroArrowLeftEndOnRectangleSolid",
  "heroArrowLeftOnRectangleSolid",
  "heroArrowLeftStartOnRectangleSolid",
  "heroArrowLeftSolid",
  "heroArrowLongDownSolid",
  "heroArrowLongLeftSolid",
  "heroArrowLongRightSolid",
  "heroArrowLongUpSolid",
  "heroArrowPathRoundedSquareSolid",
  "heroArrowPathSolid",
  "heroArrowRightCircleSolid",
  "heroArrowRightEndOnRectangleSolid",
  "heroArrowRightOnRectangleSolid",
  "heroArrowRightStartOnRectangleSolid",
  "heroArrowRightSolid",
  "heroArrowSmallDownSolid",
  "heroArrowSmallLeftSolid",
  "heroArrowSmallRightSolid",
  "heroArrowSmallUpSolid",
  "heroArrowTopRightOnSquareSolid",
  "heroArrowTrendingDownSolid",
  "heroArrowTrendingUpSolid",
  "heroArrowUpCircleSolid",
  "heroArrowUpLeftSolid",
  "heroArrowUpOnSquareStackSolid",
  "heroArrowUpOnSquareSolid",
  "heroArrowUpRightSolid",
  "heroArrowUpTraySolid",
  "heroArrowUpSolid",
  "heroArrowUturnDownSolid",
  "heroArrowUturnLeftSolid",
  "heroArrowUturnRightSolid",
  "heroArrowUturnUpSolid",
  "heroArrowsPointingInSolid",
  "heroArrowsPointingOutSolid",
  "heroArrowsRightLeftSolid",
  "heroArrowsUpDownSolid",
  "heroAtSymbolSolid",
  "heroBackspaceSolid",
  "heroBackwardSolid",
  "heroBanknotesSolid",
  "heroBars2Solid",
  "heroBars3BottomLeftSolid",
  "heroBars3BottomRightSolid",
  "heroBars3CenterLeftSolid",
  "heroBars3Solid",
  "heroBars4Solid",
  "heroBarsArrowDownSolid",
  "heroBarsArrowUpSolid",
  "heroBattery0Solid",
  "heroBattery100Solid",
  "heroBattery50Solid",
  "heroBeakerSolid",
  "heroBellAlertSolid",
  "heroBellSlashSolid",
  "heroBellSnoozeSolid",
  "heroBellSolid",
  "heroBoltSlashSolid",
  "heroBoltSolid",
  "heroBookOpenSolid",
  "heroBookmarkSlashSolid",
  "heroBookmarkSquareSolid",
  "heroBookmarkSolid",
  "heroBriefcaseSolid",
  "heroBugAntSolid",
  "heroBuildingLibrarySolid",
  "heroBuildingOffice2Solid",
  "heroBuildingOfficeSolid",
  "heroBuildingStorefrontSolid",
  "heroCakeSolid",
  "heroCalculatorSolid",
  "heroCalendarDaysSolid",
  "heroCalendarSolid",
  "heroCameraSolid",
  "heroChartBarSquareSolid",
  "heroChartBarSolid",
  "heroChartPieSolid",
  "heroChatBubbleBottomCenterTextSolid",
  "heroChatBubbleBottomCenterSolid",
  "heroChatBubbleLeftEllipsisSolid",
  "heroChatBubbleLeftRightSolid",
  "heroChatBubbleLeftSolid",
  "heroChatBubbleOvalLeftEllipsisSolid",
  "heroChatBubbleOvalLeftSolid",
  "heroCheckBadgeSolid",
  "heroCheckCircleSolid",
  "heroCheckSolid",
  "heroChevronDoubleDownSolid",
  "heroChevronDoubleLeftSolid",
  "heroChevronDoubleRightSolid",
  "heroChevronDoubleUpSolid",
  "heroChevronDownSolid",
  "heroChevronLeftSolid",
  "heroChevronRightSolid",
  "heroChevronUpDownSolid",
  "heroChevronUpSolid",
  "heroCircleStackSolid",
  "heroClipboardDocumentCheckSolid",
  "heroClipboardDocumentListSolid",
  "heroClipboardDocumentSolid",
  "heroClipboardSolid",
  "heroClockSolid",
  "heroCloudArrowDownSolid",
  "heroCloudArrowUpSolid",
  "heroCloudSolid",
  "heroCodeBracketSquareSolid",
  "heroCodeBracketSolid",
  "heroCog6ToothSolid",
  "heroCog8ToothSolid",
  "heroCogSolid",
  "heroCommandLineSolid",
  "heroComputerDesktopSolid",
  "heroCpuChipSolid",
  "heroCreditCardSolid",
  "heroCubeTransparentSolid",
  "heroCubeSolid",
  "heroCurrencyBangladeshiSolid",
  "heroCurrencyDollarSolid",
  "heroCurrencyEuroSolid",
  "heroCurrencyPoundSolid",
  "heroCurrencyRupeeSolid",
  "heroCurrencyYenSolid",
  "heroCursorArrowRaysSolid",
  "heroCursorArrowRippleSolid",
  "heroDevicePhoneMobileSolid",
  "heroDeviceTabletSolid",
  "heroDocumentArrowDownSolid",
  "heroDocumentArrowUpSolid",
  "heroDocumentChartBarSolid",
  "heroDocumentCheckSolid",
  "heroDocumentDuplicateSolid",
  "heroDocumentMagnifyingGlassSolid",
  "heroDocumentMinusSolid",
  "heroDocumentPlusSolid",
  "heroDocumentTextSolid",
  "heroDocumentSolid",
  "heroEllipsisHorizontalCircleSolid",
  "heroEllipsisHorizontalSolid",
  "heroEllipsisVerticalSolid",
  "heroEnvelopeOpenSolid",
  "heroEnvelopeSolid",
  "heroExclamationCircleSolid",
  "heroExclamationTriangleSolid",
  "heroEyeDropperSolid",
  "heroEyeSlashSolid",
  "heroEyeSolid",
  "heroFaceFrownSolid",
  "heroFaceSmileSolid",
  "heroFilmSolid",
  "heroFingerPrintSolid",
  "heroFireSolid",
  "heroFlagSolid",
  "heroFolderArrowDownSolid",
  "heroFolderMinusSolid",
  "heroFolderOpenSolid",
  "heroFolderPlusSolid",
  "heroFolderSolid",
  "heroForwardSolid",
  "heroFunnelSolid",
  "heroGifSolid",
  "heroGiftTopSolid",
  "heroGiftSolid",
  "heroGlobeAltSolid",
  "heroGlobeAmericasSolid",
  "heroGlobeAsiaAustraliaSolid",
  "heroGlobeEuropeAfricaSolid",
  "heroHandRaisedSolid",
  "heroHandThumbDownSolid",
  "heroHandThumbUpSolid",
  "heroHashtagSolid",
  "heroHeartSolid",
  "heroHomeModernSolid",
  "heroHomeSolid",
  "heroIdentificationSolid",
  "heroInboxArrowDownSolid",
  "heroInboxStackSolid",
  "heroInboxSolid",
  "heroInformationCircleSolid",
  "heroKeySolid",
  "heroLanguageSolid",
  "heroLifebuoySolid",
  "heroLightBulbSolid",
  "heroLinkSolid",
  "heroListBulletSolid",
  "heroLockClosedSolid",
  "heroLockOpenSolid",
  "heroMagnifyingGlassCircleSolid",
  "heroMagnifyingGlassMinusSolid",
  "heroMagnifyingGlassPlusSolid",
  "heroMagnifyingGlassSolid",
  "heroMapPinSolid",
  "heroMapSolid",
  "heroMegaphoneSolid",
  "heroMicrophoneSolid",
  "heroMinusCircleSolid",
  "heroMinusSmallSolid",
  "heroMinusSolid",
  "heroMoonSolid",
  "heroMusicalNoteSolid",
  "heroNewspaperSolid",
  "heroNoSymbolSolid",
  "heroPaintBrushSolid",
  "heroPaperAirplaneSolid",
  "heroPaperClipSolid",
  "heroPauseCircleSolid",
  "heroPauseSolid",
  "heroPencilSquareSolid",
  "heroPencilSolid",
  "heroPhoneArrowDownLeftSolid",
  "heroPhoneArrowUpRightSolid",
  "heroPhoneXMarkSolid",
  "heroPhoneSolid",
  "heroPhotoSolid",
  "heroPlayCircleSolid",
  "heroPlayPauseSolid",
  "heroPlaySolid",
  "heroPlusCircleSolid",
  "heroPlusSmallSolid",
  "heroPlusSolid",
  "heroPowerSolid",
  "heroPresentationChartBarSolid",
  "heroPresentationChartLineSolid",
  "heroPrinterSolid",
  "heroPuzzlePieceSolid",
  "heroQrCodeSolid",
  "heroQuestionMarkCircleSolid",
  "heroQueueListSolid",
  "heroRadioSolid",
  "heroReceiptPercentSolid",
  "heroReceiptRefundSolid",
  "heroRectangleGroupSolid",
  "heroRectangleStackSolid",
  "heroRocketLaunchSolid",
  "heroRssSolid",
  "heroScaleSolid",
  "heroScissorsSolid",
  "heroServerStackSolid",
  "heroServerSolid",
  "heroShareSolid",
  "heroShieldCheckSolid",
  "heroShieldExclamationSolid",
  "heroShoppingBagSolid",
  "heroShoppingCartSolid",
  "heroSignalSlashSolid",
  "heroSignalSolid",
  "heroSparklesSolid",
  "heroSpeakerWaveSolid",
  "heroSpeakerXMarkSolid",
  "heroSquare2StackSolid",
  "heroSquare3Stack3dSolid",
  "heroSquares2x2Solid",
  "heroSquaresPlusSolid",
  "heroStarSolid",
  "heroStopCircleSolid",
  "heroStopSolid",
  "heroSunSolid",
  "heroSwatchSolid",
  "heroTableCellsSolid",
  "heroTagSolid",
  "heroTicketSolid",
  "heroTrashSolid",
  "heroTrophySolid",
  "heroTruckSolid",
  "heroTvSolid",
  "heroUserCircleSolid",
  "heroUserGroupSolid",
  "heroUserMinusSolid",
  "heroUserPlusSolid",
  "heroUserSolid",
  "heroUsersSolid",
  "heroVariableSolid",
  "heroVideoCameraSlashSolid",
  "heroVideoCameraSolid",
  "heroViewColumnsSolid",
  "heroViewfinderCircleSolid",
  "heroWalletSolid",
  "heroWifiSolid",
  "heroWindowSolid",
  "heroWrenchScrewdriverSolid",
  "heroWrenchSolid",
  "heroXCircleSolid",
  "heroXMarkSolid",
]
`;
